src/main/resources/application.yaml

```diff
@@ -74,7 +74,7 @@ idam:
       client:
         baseUrl: ${S2S_BASE_URI:http://localhost:4502}
   s2s-authorised:
-    services: ${S2S_NAMES_WHITELIST:ccd_gw,em_gw,em_hrs_ingestor}
+    services: ${S2S_NAMES_WHITELIST:ccd_gw,em_gw,em_hrs_ingestor,em_hrs_api}
```

Comment:
why are we whitelisting ourselves, or is that what the ingestor is going to use to call hrs-api